integrations/cypress/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from '@bigtest/interactor';
+import './cypress';
```

Comment:
what is the import doing here?

Reply:
Isn't it necessary to import the functions into index so that they're applied in the environment that people install @bigtest/cypress into?

Reply:
oh right yes, I forgot about all that `Cypress.Commands.add` business

Can we move this into a separate .d.ts file

```ts
declare global {
  namespace Cypress {
    interface Chainable<Subject> {
      do(interaction: Interaction<void> | Interaction<void>[]): Chainable<Subject>;
      expect(interaction: ReadonlyInteraction<void> | ReadonlyInteraction<void>[]): Chainable<Subject>;
    }
  }
}
```

Also, can we not use 

```json
    "types": [
      "cypress",
      "node"
    ]
  ```
rather than

```ts
/// <reference types="cypress" />
```

Reply:
I can give it a try but I think we've run into this problem before and ended up going in circles (for both separating the global declaration and avoiding reference types).

Reply:
@minkimcello I can do it on a separate PR if it proves problematic.  Don't waste too much time on it

Reply:
I can't think of any reason why it should not work but maybe I'm missing something.

Reply:
oh okay 👌 so i'll resolve this conversation